README.md

```diff
@@ -43,6 +43,24 @@ already have these, if not please download and run vcredist.x64.exe from
 [Microsoft](https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads) to install them prior to running
 cbmc.
 
+### Linux
+
+For different linux environments, you have these choices:
+
+1. Install CBMC through the distribution's repositories, with the downside
+   that this might install an older version of cbmc, depending on what the
+   package maintenance policy of the distribution is, or
+2. Install CBMC through the `.deb` package built by each release, available
+   on the [releases](https://github.com/diffblue/cbmc/releases) page. To
+   do that, download the `.deb` package and run `apt install cbmc-x.y.deb`
```

Comment:
Nit: more stable version of this is `dpkg -i`, but if `apt install` also works I guess it works.

Reply:
I think `apt` is always preferable, as it's a higher level tool and can do dependency management better than `dpkg`.